cogs/commands.py

```diff
@@ -379,6 +379,29 @@ async def suggestion(self, ctx, *, suggestion: str):
         msg = await ctx.send(embed=em)
         await msg.add_reaction('👍')
         await msg.add_reaction('👎')
+        
+    @suggestion.command(name='edit')
+    async def edit_suggestion(self, ctx, message: commands.MessageConverter, *, suggestion: str):
+        """Edit an existing poll/suggestion. You will have to rewrite the suggestion."""        
+        try:
+            if message.author != ctx.bot.user:
+                if not message.embeds[0].author.name.startswith("Poll"):
+                    return await ctx.send("That message is not a poll!")
+            else:
+                em = message.embeds[0]
+                if is_mod(ctx):
+                    em.description = suggestion
+                    await message.edit(embed=em)
+                    return await ctx.send("Successfully edited suggestion.")
+                if message.edited_at is not None or datetime.now().replace(microsecond=0)-message.created_at.replace(microsecond=0) > timedelta(minutes=5): # If it has been edited yet, or was sent more than 5 minutes ago
```

Comment:
Just check if the suggestion is more than 5 minutes old.
And the reply should say why you can't edit